lib/filesystem_utils.pm

```diff
@@ -275,12 +275,21 @@ sub format_partition {
     if ($filesystem =~ /ext4/) {
         $options = $args{options} // "-F";
     }
+    elsif ($filesystem =~ /ocfs2/) {
+        $options = $args{options} // "-F --fs-features=local --fs-feature-level=max-features";
+    }
     else {
         $options = $args{options} // "-f";
     }
     script_run("umount -f $part");
     sleep 1;
-    assert_script_run("mkfs.$filesystem $options $part");
+    if ($filesystem =~ /ocfs2/) {
+        # mkfs.ocfs2 will still require input y even you used -F
+        assert_script_run("y | mkfs.$filesystem $options $part");
```

Comment:
I guess there should be `echo y | mkfs.$filesystem $options $part`,
or else
`yes | mkfs.$filesystem $options $part`
But IMO `yes` is better, because it output 'y' repeatedly.

Reply:
The input only requires y here,`mkfs.ocfs2 -F --fs-features=local --fs-feature-level=max-features /home/test_dev ;
mkfs.ocfs2 1.8.7
Overwriting existing ocfs2 partition.
Proceed (y/N): `

Reply:
change to echo is better, I'll change this. It will solve the problem like this http://10.67.133.133/tests/249#step/run/478

Reply:
I tried with echo and script_run can't solve this problem in http://10.67.133.133/tests/249#step/run/478. I'll revert this to the previous background_script_run and type y solution.